File: app/src/main/java/com/example/alkewalletandroid/MainActivity5.java
package com.example.alkewalletandroid;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.ImageView;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;
import androidx.core.graphics.Insets;
import androidx.core.view.ViewCompat;
import androidx.core.view.WindowInsetsCompat;

public class MainActivity5 extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main5);;

        ImageView perfil = findViewById(R.id.img_perfil);
        Button enviar = findViewById(R.id.enviar_dinero);
        Button ingresar = findViewById(R.id.ingresar_dinero);

        perfil.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity5.this, MainActivity9.class);
                startActivity(intent);
            }
        });

        enviar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity5.this, MainActivity8.class);
                startActivity(intent);
            }
        });

        ingresar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity5.this, MainActivity7.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/alkewalletandroid/MainActivity5.kt
package com.example.alkewalletandroid

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import androidx.appcompat.app.AppCompatActivity

class MainActivity5 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main5)
        val perfil = findViewById<ImageView>(R.id.img_perfil)
        val enviar = findViewById<Button>(R.id.enviar_dinero)
        val ingresar = findViewById<Button>(R.id.ingresar_dinero)
        perfil.setOnClickListener {
            val intent = Intent(this@MainActivity5, MainActivity9::class.java)
            startActivity(intent)
        }
        enviar.setOnClickListener {
            val intent = Intent(this@MainActivity5, MainActivity8::class.java)
            startActivity(intent)
        }
        ingresar.setOnClickListener {
            val intent = Intent(this@MainActivity5, MainActivity7::class.java)
            startActivity(intent)
        }
    }
}